docs/src/administration/web/configure-environment.md

```diff
@@ -50,84 +51,86 @@ There are also additional options:
+Under **HTTP access control**, you can [control access to your environment using HTTP methods](../../environments/http-access-control.md):
 
-## Environment settings
+![Settings control access with password and by IP](/images/management-console/settings-basics-access-control.png "0.5")
 
-To see settings for a given environment, click {{< icon settings >}} **Settings** within that environment.
+### Variables
 
-![Settings for an environment](/images/management-console/env-settings.png "0.75")
+Under **Variables**, you can define [environment variables](../../development/variables/_index.md):
 
-### Environment name
+![Configure Platform.sh environment variables](/images/management-console/settings-variables-environment.png "0.6")
 
-The first setting allows you to modify the name of the environment and view its parent environment.
+## Service information
 
-### Status
+For each environment, you can view information about how your routes, services, and apps are currently configured.
 
-The **Status** setting shows whether or not your environment is [active](../../other/glossary.md#active-environment).
-For non-Production environment, you can [change their status](../../environments/deactivate-environment.md).
+To do so, click **Services**.
+By default, you see configured routes.
 
-![Environment status](/images/management-console/env-status.png "0.5")
+### Routes
 
-### Outgoing emails
+The **Router** section shows a list of all the routes configured on your environment.
+You can see each route's type, and check if caching and server side includes have been enabled for it:
 
-You can allow your environment to [send outgoing emails](../../development/email.md).
+![Services: routes](/images/management-console/service-tab/routes.png "0.5")
 
-![Environment email](/images/management-console/env-email.png "0.75")
+To view the `.platform/routes.yaml` file where your routes are set up, click **Configuration**.
 
-### Search engine visibility
+### Applications
 
-This setting enables you to tell [search engines to ignore the site](../../environments/search-engine-visibility.md).
+To see detailed information about an app container, select it in the tree or list on the left-hand side:
 
-![Environment search](/images/management-console/env-search.png "0.5")
+![Services: app overview](/images/management-console/service-tab/app-overview.png "0.5")
 
-### HTTP access control
+The **Overview** tab gives you information about your app.
+You can see the language version, the container size, the amount of persistent disk,
+the number of active workers and cron jobs, and the command to SSH into the container.
+You can also see all cron jobs with their name, frequency, and command.
 
-This setting enables you to [control access using HTTP methods](../../environments/http-access-control.md).
+To view the `.platform/app.yaml` file where your app is set up, click **Configuration**.
```

Comment:
:lady_beetle: The reason this wasn't `.platform.app.yaml` before is because there are multiple ways to define app configurations, including a `.platform/applications.yaml` file.

It might be best to leave them all as just `configuration file` or something generic like that. I'm not sure that people need to see the file name here. It might just be extra information.

Reply:
Ok I changed them all to "configuration file" then.